assets/package-test/apt.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Will these files be removed from teleport.e?

Reply:
when we will able to deprecate upgrader script - yes, they not fully identical